Google.Api.Generator.Tests/ProtoTests/RoutingHeadersExplicit/RoutingHeadersExplicit.proto

```diff
@@ -0,0 +1,126 @@
+﻿syntax = "proto3";
+
+package testing.routingheadersexplicit;
+
+option csharp_namespace = "Testing.RoutingHeadersExplicit";
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/routing.proto";
+
+service RoutingHeadersExplicit {
+  option (google.api.default_host) = "routingheaders.example.com";
+
+  rpc NoRouting(SimpleRequest) returns(Response);
+
+  rpc PlainNoTemplate(SimpleRequest) returns(Response) {
+    option (google.api.http) = {
+      get: "/v2/{name=projects/*/instances/*/tables/*}:sampleRowKeys"
+    };
+    option (google.api.routing) = {
+      routing_parameters {
+        field: "name"
+      }
+    };
+  }
+
+  rpc PlainNoExtraction(SimpleRequest) returns(Response) {
```

Comment:
I missed this before - please fix the indentation. (Again, that could be post merge.)

Reply:
I was not sure what exactly was wrong with the indentation. Reformatted the file.